README.md

```diff
@@ -163,26 +163,33 @@ python run_evals_accelerate.py \
     --output_dir output_dir
 ```
 
-### Evaluate a model on community submitted/custom tasks.
+### Evaluate a model on extended, community, or custom tasks.
 
-You can use `lighteval` to evaluate models on custom or community submitted tasks. Select your task of interest (which might have its own requirements to install first), and run:
+You can use `lighteval` to evaluate models on tasks that require special processing or those that have been added by the community. These tasks have their own evaluation suite and are defined as follows:
+
+* `extended`: tasks which have complex pre- or post-processing and are added by the `lighteval` maintainers. See the [`extended_tasks`](./extended_tasks) folder for examples.
+* `community`: tasks which have been added by the community. See the [`community_tasks`](./community_tasks) folder for examples.
+* `custom`: tasks which are defined locally and not present in the core library. Use this suite if you want to use a special metric or task together with the `lighteval` features.
```

Comment:
`Use this suite if you want to experiment with designing your special metric... ?`